### PR TITLE
Doc - Modify Dispatcher to use simplified thread-pool-config values, …

### PR DESCRIPTION
**Pull request by patriknw**
…as described by @ktoso in the the issue (#21572)

### CODE REVIEW

**Comment by patriknw** [comment]
Refs #21572

File: akka-docs/rst/scala/dispatchers.rst [hunk 1]
```diff
@@ -153,6 +130,10 @@ And then using it:
 
 .. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#defining-fixed-pool-size-dispatcher
 
+Another example that uses the thread pool based on the number of cores (e.g. for CPU bound tasks)
+
+.. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#my-thread-pool-dispatcher-config
```
Comment (ktoso) [inline akka-docs/rst/scala/dispatchers.rst]: Hmmm, but that's still the complex example though?
https://github.com/akka/akka/blob/master/akka-docs/rst/scala/code/docs/dispatcher/DispatcherDocSpec.scala#L86

Comment (ktoso) [inline akka-docs/rst/scala/dispatchers.rst]: The idea was to show:


```
    fixed-pool-size = 16
```

Comment (patriknw) [inline akka-docs/rst/scala/dispatchers.rst]: This is at the bottom of the page in the section "More dispatcher configuration examples". There we also show the `fixed-pool-size = 16`, and at the top of the page we now show the `fixed-pool-size = 16` example.

File: akka-docs/rst/java/dispatchers.rst [hunk 2]
```diff
@@ -40,9 +40,9 @@ is to configure the dispatcher:
   threads the pool keep running in order to reduce the latency of handling a new incoming task.
   You can read more about parallelism in the JDK's `ForkJoinPool documentation`_.
 
-And here's another example that uses the "thread-pool-executor":
-
-.. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#my-thread-pool-dispatcher-config
+another example that uses the "thread-pool-executor":
```
Comment (johanandren) [inline akka-docs/rst/java/dispatchers.rst]: Big-A for start of sentence

File: akka-docs/rst/scala/dispatchers.rst [hunk 3]
```diff
@@ -40,9 +40,9 @@ is to configure the dispatcher:
   threads the pool keep running in order to reduce the latency of handling a new incoming task.
   You can read more about parallelism in the JDK's `ForkJoinPool documentation`_.
 
-And here's another example that uses the "thread-pool-executor":
-
-.. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#my-thread-pool-dispatcher-config
+another example that uses the "thread-pool-executor":
+ 
+ .. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#fixed-pool-size-dispatcher-config
```
Comment (johanandren) [inline akka-docs/rst/scala/dispatchers.rst]: Intended indent of one space, or is it the other ones that are missing the indent?

**Review by johanandren — APPROVED** [review]
LGTM with some minimal things.

File: akka-docs/rst/scala/dispatchers.rst (hunk 3 above)
Comment (patriknw) [inline akka-docs/rst/scala/dispatchers.rst]: my bad

**Comment by patriknw** [comment]
fixed comments

File: akka-docs/rst/scala/dispatchers.rst (hunk 1 above)
Comment (ktoso) [inline akka-docs/rst/scala/dispatchers.rst]: Ok that's all good then :) Thanks!

**Pull request merged**